backend/tests/test_author_api.py

```diff
@@ -8,6 +8,15 @@
 class TestAuthorAPI:
 
     def test_get_profile_by_author_id(self, client, test_user, friend_user):
+        '''
+            Checking scenario where the user does not exist, 404 Error
+        '''
+        nouserResponse = client.get('/author/{}/'.format("https://cmput404-socialdistribution.herokuapp.com/author/20"))
```

Comment:
Change this id to something more “wrong" in case the user actually exists in the db

Reply:
Done!